src/api.ts

```diff
@@ -0,0 +1,8 @@
+export type ICheckDisable = {
```

Comment:
In general I don't like very much this thing to start interfaces with I, what's the point of doing this? Also this is not even an interface but a type, why I?